desktop/app/index.js

```diff
@@ -55,9 +53,13 @@ installer(err => {
   splash()
 })
 
-if (app.dock && !mw.initiallyVisible) {
-  app.dock.hide()
-}
+app.once('ready', () => {
+  const mw = mainWindow()
```

Comment:
There's a potential race condition where we try to show the main window before the app is ready.